src/components/ClusterOverview/Details/storageDetails.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+
+import { ClusterOverviewContextGenericConsumer } from '../ClusterOverviewContext';
+import { InlineLoading } from '../../Loading';
+import { Details } from './Details';
+
+export const StorageDetails = ({LoadingComponent }) => {
+  const items = {
+    name: {
+      title: 'Name',
+      value: 'Cluster 1',
```

Comment:
we can get cluster name from infrastructure resource. See `ClusterDetails` component.

Reply:
@rawagner I think this is a ceph cluster name and will be different from Openshift cluster name.

Reply:
@cloudbehl ah, ok! My bad